src/vconsole/vconsole-setup.c

```diff
@@ -223,9 +206,17 @@ static int font_load_and_wait(const char *vc, const char *font, const char *map,
  * processing, stty iutf8
  */
 static void setup_remaining_vcs(int fd, int vc_max, bool utf8) {
+        struct console_font_op cfo = {
+                .op = KD_FONT_OP_GET, .flags = 0,
+                .width = 32, .height = 32,
+                .charcount = 512,
+        };
         struct vt_stat vcs = {};
+        struct unimapinit adv = {};
         struct unimapdesc unimapd;
         _cleanup_free_ struct unipair* unipairs = NULL;
+        _cleanup_free_ unsigned char *fontbuf = NULL;
```

Comment:
fontbuf is allocated and assigned to cfo.data which is of type unsigned char* (see struct console_font_op cfo)